src/DiceRoller/Dice.cpp

```diff
@@ -8,6 +8,7 @@
 #include "Dice.hpp"
 #include <iostream>
 #include <vector>
+#include <algorithm>
```

Comment:
@derkanadier , I believe this was missing from your commit. You call the  `sort` function in your class, which is from this library, please confirm the modification.

Reply:
I removed it at some point because the sorting still seemed to work fine without that _include_. However, if you think it's best to add it back, then go ahead :)